x/gov/types/codec.capnp

```diff
@@ -0,0 +1,45 @@
+ @0xf6e87acf2c3fc2e7;
+
+ using Go = import "/go.capnp";
+ $Go.package("types");
+ $Go.import("github.com/cosmos/cosmos-sdk/x/gov/types");
+
+ using SDK = import "/types/codec.capnp";
+
+struct Msg(Content) {
```

Comment:
Shouldn't this be a union type?
As written, it seems like it may contain a deposit, a proposal, and a vote.
Is that wanted?